astropy/stats/jackknife.py

```diff
@@ -39,7 +40,7 @@ def jackknife_resampling(data):
         Stanford University, December, 1980.
 
     .. [3] Jackknife resampling <https://en.wikipedia.org/wiki/Jackknife_resampling>
-    """
+    """  # noqa
```

Comment:
why the noqa?

Reply:
The "Jackknife resampling" URL is too long. heh heh heh

Reply:
oh, but it's only 84 character, there are plenty of longer lines around, just in stats 42

Reply:
If this really bothers you, I can back it out. This was the only thing linter was complaining about for this file, so I took liberties.

Reply:
no worries, I would be more than happy to go 80, or decide in a middle ground 100, but then stick to it.